geometry_msgs/Point[] frontiers
geometry_msgs/Point[] clusters
std_msgs/Int32 map_increase
